### PR TITLE
ideas for working around the integrity checker issues

### PR DESCRIPTION
**Pull request by jnweiger**
food for thought only. 
DO NOT MERGE

### CODE REVIEW

**Comment by phil-davis** [comment]
https://github.com/owncloud/core/blob/master/lib/private/IntegrityCheck/Checker.php#L154

I suspect that `generateHashes` has trouble doing hashes for the file names with Unicode in them. It has various `strlen` `substr` `trim` that seem to work OK, but maybe something in there gets messed up for file names with Unicode. And so the integrity-checker has no hash to compare for those files.

File: .drone.star [hunk 1]
```diff
@@ -273,6 +273,10 @@ def docker(config):
               'OWNCLOUD_DB_USERNAME': 'owncloud',
               'OWNCLOUD_DB_PASSWORD': 'owncloud',
               'OWNCLOUD_DB_NAME': 'owncloud',
+              # $config['integrity.excluded.files'] = explode(',', getenv('OWNCLOUD_INTEGRITY_EXCLUDED_FILES'));
+              'OWNCLOUD_INTEGRITY_EXCLUDED_FILES': 'something',
+              # $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISBLED') == 'true';
+              'OWNCLOUD_INTEGRITY_CHECK_DISABLED': 'true',
```
Comment (phil-davis) [inline .drone.star:279]: This seems a good thing to do - this is the pipeline just for UI tests. The server of those pipelines will be "thrown away" at the end of the test run. So the setting will not end up in the published images.

Comment (micbar) [reply .drone.star:279]: We will use an occ command.
I am not eager to add this variable to the official base image.

Comment (phil-davis) [reply .drone.star:279]: I saw your commit in #204 - that is easy and good, the setting only happens directly in the UI pipelines.

Comment (jnweiger) [reply .drone.star:279]: ok. we agree that disabling the integrity checker is a good thing here.
Now we need to find a syntax that allows us to do that. 
the syntax here does not wor. The variable OWNCLOUD_INTEGRITY_CHECK_DISABLED is my private idea, it does not exist in the base images.

**Pull request closed without merging**